postgres/assets/configuration/spec.yaml

```diff
@@ -427,6 +427,22 @@ files:
           value:
             type: number
             example: 3500
+    - name: database_metadata
+      description: Configure collection of pg_settings. This is an alpha feature.
+      options:
+        - name: collect_settings
+          description: |
+            Enable collection of pg_settings. Requires `dbm: true`.
+          value:
+            type: boolean
+            example: false
+        - name: settings_collection_interval
+          description: |
+            Set the database settings collection interval (in seconds). Each collection involves a single query to
+            `pg_settings`.
+          value:
+            type: number
+            example: 600
```

Comment:
Let's make these hidden to avoid early configuration and customers copying configs and messing themselves up later down the road.

Reply:
> Let's make these hidden to avoid early configuration and customers copying configs and messing themselves up later down the road.

if we do this https://github.com/DataDog/integrations-core/pull/14577#discussion_r1200852950, why do you assume we are going to change this? also its an alpha feature so seems fair that we could break the config at some point if we want to?